Classes/Controller/CartController.php

```diff
@@ -629,6 +629,7 @@ protected function sendBuyerMail(
         \Extcode\Cart\Domain\Model\Order\Address $billingAddress,
         \Extcode\Cart\Domain\Model\Order\Address $shippingAddress = null
     ) {
+	    /* @var \Extcode\Cart\Service\MailHandler $mailHandler*/
```

Comment:
Please indent with spaces:
https://docs.typo3.org/typo3cms/CodingGuidelinesReference/PhpFileFormatting/GeneralRequirementsForPhpFiles/Index.html#whitespace-and-indentation

Please indent comments with the same number of spaces:
https://docs.typo3.org/typo3cms/CodingGuidelinesReference/PhpFileFormatting/PhpSyntaxFormatting/Index.html#comments